src/jane/waveforms/models.py

```diff
@@ -235,9 +235,17 @@ class Restriction(models.Model):
 
     Waveforms are generally seen as public if not listed here.
     """
-    network = models.CharField(max_length=2, db_index=True)
-    station = models.CharField(max_length=5, db_index=True)
-    users = models.ManyToManyField(User, db_index=True)
+    help_text = ('Use a single asterisk/star (*) to affect all station '
+                 'codes. Use a single asterisk/star in both fields, to '
+                 'restrict all network/station code combinations.')
+    network = models.CharField(max_length=2, db_index=True,
+                               help_text=help_text)
+    station = models.CharField(max_length=5, db_index=True,
+                               help_text=help_text)
+    users = models.ManyToManyField(
+        User, db_index=True,
```

Comment:
I'm not sure if this has to be indexed - each index slows down the DB and I assume the other restrictions would already cut down the query enough so the user queries would be cheap? Not sure to be honest - might be worth to benchmark for a large database but the waveform table will be by far the biggest one so it has to reasonably fast.

Reply:
I didnt change the indexing, just added the help text.

Reply:
btw: changing the ```help_text``` requires a ```makemigrations``` too - even if it doesn't touch the database schema

Reply:
huh.. on my local jane the help text shows up immediately..

Reply:
sure - its working right away - but I bet you get also a new warning that a schemamigration is required ...

Reply:
where should this warning pop up? at least in the debug mode runserver I dont see anything..

Reply:
```python manage.py migrate```

should show something like
```
  Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

Reply:
Please also commit the migrations.

Reply:
tbh, the whole migrations stuff is still beyond me, I actually get errors due to some plugin changes when running `python manage.py migrate`, so hard to tell about warnings